packages/cli/src/Generate.ts

```diff
@@ -255,21 +256,10 @@ Please make sure they have the same version.`
         if (hideHints) {
           hint = `${getHardcodedUrlWarning(config)}${breakingChangesStr}${versionsWarning}`
         } else {
-          const tryAccelerateMessage = `Deploying your app to serverless or edge functions?
-Try Prisma Accelerate for connection pooling and caching.
-${link('https://pris.ly/cli/--accelerate')}`
-
-          const boxedTryAccelerateMessage = drawBox({
-            height: tryAccelerateMessage.split('\n').length,
-            width: 0, // calculated automatically
-            str: tryAccelerateMessage,
-            horizontalPadding: 2,
-          })
-
           hint = `
-Start by importing your prisma client (See:  http://pris.ly/d/importing-client)
+Start by importing your Prisma Client (See: http://pris.ly/d/importing-client)
 
-${boxedTryAccelerateMessage}
+${promotion.text} ${promotion.link}
 ${getHardcodedUrlWarning(config)}${breakingChangesStr}${versionsWarning}`
```

Comment:
```suggestion
${promotion.text} ${promotion.link}${getHardcodedUrlWarning(config)}${breakingChangesStr}${versionsWarning}`
```

Reply:
and then update `getHardcodedUrlWarning` like so:
```diff
export function getHardcodedUrlWarning(config: ConfigMetaFormat) {
  const datasource = config.datasources?.[0]

  if (
    datasource !== undefined &&
    datasource.provider !== 'sqlite' &&
    (datasource.url.fromEnvVar === null || datasource.directUrl?.fromEnvVar === null)
  ) {
-    return `\n🛑 Hardcoding URLs in your schema poses a security risk: ${link('https://pris.ly/d/datasource-env')}\n`
+    return `\n\n🛑 Hardcoding URLs in your schema poses a security risk: ${link('https://pris.ly/d/datasource-env')}\n`
  }

  return ''
}
```

Reply:
This removes an extra newline when this hardcoded url doesn't exist

Reply:
```
${promotion.text} 
${promotion.link}
```

This would put the link bellow the line. Why do you suggest this change? 

I haven't replicated this scenario to see how it looks in the cli

Reply:
Ah, no, it doesn't they're on the same line, that's just how it got rendered on github

Reply:
<img width="882" alt="image" src="https://github.com/user-attachments/assets/a9fb7fda-99d8-4899-b392-5a44e9f40a64">